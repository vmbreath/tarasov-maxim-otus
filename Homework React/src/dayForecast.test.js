import React from 'react';
import renderer from 'react-test-renderer';
import DayForecast from './DayForecast';

test('DayForecast renders exactly', () => {
    const component = renderer.create(
        <DayForecast town={'Москва'}
                     feelsLike={666}
                     temperature={666}
                     description={'Ясно'}
                     icon={'01n'}
                     windSpeed={666}
                     date={'0.0.0'}
                     snow={666}
                     rain={666}/>
    );
    let tree = component.toJSON();
    expect(tree).toMatchSnapshot();

});