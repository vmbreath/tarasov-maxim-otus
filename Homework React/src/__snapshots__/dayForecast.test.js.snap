// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`DayForecast renders exactly 1`] = `
<div
  className="css-z7ntyl-container"
>
  <h3
    className="css-1j4cfgk-style"
  >
    Прогноз на 
    0.0.0
  </h3>
  <p>
    Город: 
    Москва
     
  </p>
  <p>
    Чувствуется как: 
    666
     °C
  </p>
  <p>
    Температура: 
    666
     °C
  </p>
  <p>
    Объем дождя за последние 3 часа: 
    666
     мм
  </p>
  <p>
    Объем снега за последние 3 часа: 
    666
     мм
  </p>
  <p>
    Ясно
  </p>
  <img
    src="http://openweathermap.org/img/w/01n.png"
  />
  <p>
    Скорость ветра: 
    666
     м/с
  </p>
</div>
`;

exports[`Renders exactly 1`] = `
<div
  className="css-z7ntyl-container"
>
  <h3
    className="css-1j4cfgk-style"
  >
    Прогноз на 
    0.0.0
  </h3>
  <p>
    Город: 
    Москва
     
  </p>
  <p>
    Чувствуется как: 
    666
     °C
  </p>
  <p>
    Температура: 
    666
     °C
  </p>
  <p>
    Объем дождя за последние 3 часа: 
    666
     мм
  </p>
  <p>
    Объем снега за последние 3 часа: 
    666
     мм
  </p>
  <p>
    Ясно
  </p>
  <img
    src="http://openweathermap.org/img/w/01n.png"
  />
  <p>
    Скорость ветра: 
    666
     м/с
  </p>
</div>
`;
